.github/workflows/erd.yml

```diff
@@ -0,0 +1,30 @@
+name: ERD
+on:
+  pull_request:
+    paths:
+      - "db/schema.rb"
```

Comment:
here

Reply:
ohhhh, I misunderstood your comment :) got it, yeah I'll go ahead and add it as a filter for this action